Ez Namey.js

```diff
@@ -0,0 +1,9 @@
+//Startup name generator)
+//Create a function that returns a random startup name
+
+
+const firstWords = ["Juicy", "Golly", "Fwenzy", "Winky", "Sunny", "Breezy", "Slick", "Zippy", "Dizzy", "Dandy"];
+const secondWords = ["Labs", "Studios", "Solutions", "Co", "Corp", "Industries", "Enterprises", "Tech", "Web", "Net"];
+const randomNumber = Math.floor(Math.random() * 10) + 0;
+const startupName = firstWords[randomNumber] + " " + secondWords[randomNumber];
```

Comment:
This will always return the same pairs of names. Use an extra variable for another randomNumber to use for the `secondWords` array.